cocos/gfx/webgl/webgl-swapchain.ts

```diff
@@ -219,6 +220,10 @@ export class WebGLSwapchain extends Swapchain {
         return this._extensions as IWebGLExtensions;
     }
 
+    get blitManager () {
+        return this._blitManager!;
```

Comment:
`blitManager` managed by device might be more appropriate.

Reply:
Yes, we can put it in device for now.

Reply:
Good suggestion, but currently we cant, since `swapchain` manages many WebGL context states.